src/components/Dashboard.tsx

```diff
@@ -211,8 +211,12 @@ const Dashboard = ({ budget }: DashboardProps) => {
                           )}
                         </td>
                         <td
-                          className={`hidden md:table-cell whitespace-nowrap px-3 pb-3 text-sm font-semibold text-right`}
-                        ></td>
+                          className={`hidden md:table-cell whitespace-nowrap px-3 pb-3 text-sm font-semibold text-right ${
+                            budgetMonth.spent < 0 ? 'positive' : 'text-gray-500'
+                          }`}
+                        >
+                          {formatMoney(budgetMonth.spent, budget.currency)}
```

Comment:
```suggestion
                          {formatMoney(budgetMonth.spent, budget.currency, 'auto')}
```

This only adds a sign for negative numbers, which avoids cluttering the dashboard with a bunch of "+" signs and is consistent with the way we display the spent amount per envelope: https://github.com/envelope-zero/frontend/blob/dfac540370eda135d61ee611dc59ef3bc827a36a/src/components/EnvelopeMonth.tsx#L161-L167